packages/ui/src/components/filter/components/buttons/buttons.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Button } from '../../../index.js';
+import { generateAriaDescription } from '../../filter.util.js';
+
+import { styles } from './buttons.styles.js';
+import { type ButtonsProps } from './buttons.types.js';
+
+export function Buttons({
+  filterButtons,
+  onClick,
+  selectedButton,
+  resultsFound,
+  tag: Tag = 'nav',
```

Comment:
Not sure `nav` needs to be the default tag

Reply:
Just looking at Google implementation here, https://material-web.dev/components/chip/stories/